cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-scaling.adoc

```diff
@@ -0,0 +1,259 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-scaling"]
+= Tutorial: Scaling an application
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-scaling
+:source-highlighter: pygments
+:pygments-style: emacs
+:icons: font
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2024-04-10
+
+== Scaling
+
+You can manually scale your pods or you can automatically scale your pods by using the Horizontal Pod Autoscaler (HPA). You can also scale your cluster nodes.
+
+=== Manual pod scaling
+
+You can manually scale the pods and nodes of your applications:
+
+* by changing your replicaset and deployment definition 
+* by using the command line
+* by using the web console
+* by changing your *deployment* definition in the `ostoy-frontend-deployment.yaml`
+
+This workshop starts by using only one pod for the microservice. By using one pod, the Kubernetes Replication Controller strives to keep one pod alive. You can also define pod autoscaling by using the link:https://docs.openshift.com/container-platform/latest/nodes/pods/nodes-pods-autoscaling.html[Horizontal Pod Autoscaler](HPA). The HPA is based on workload, and it expands past defined limits if needed.
```

Comment:
In the second sentence I'd start with "By defining a replica of "1" in our deployment definition, the Kubernetes...".  The two idea's here don't connect well when I am reading it.  It is not clear to me how we went from saying we only defined 1 pod and how the HPA is connected.  The second part seems disconnected.  Can we try to connect the ideas?  Something like "You can also define pod autoscaling by using the Horizontal Pod Autoscaler (HPA) which is based on the load and will scale out more pods, beyond our initial definition, if high load is experienced."